ClearentIdtechIOSFramework/Wrapper/ClearentWrapper.swift

```diff
@@ -82,38 +87,30 @@ public enum UserAction: String, CaseIterable {
             return ClearentConstants.Localized.Bluetooth.turnedOff
         case .noBluetoothPermission:
             return ClearentConstants.Localized.Bluetooth.noPermission
-        }
-    }
-    
-    static func action(for text: String) -> UserAction? {
-        UserAction.allCases.first { $0.description == text }
-    }
-}
-
-public enum UserInfo: String, CaseIterable {
-    case authorizing,
-         processing,
-         goingOnline,
-         amountNotAllowedForTap,
-         chipNotRecognized
-    
-    var description: String {
-        switch self {
         case .authorizing:
             return CLEARENT_TRANSACTION_AUTHORIZING
         case .processing:
             return CLEARENT_TRANSACTION_PROCESSING
-        case .goingOnline:
-            return CLEARENT_TRANSLATING_CARD_TO_TOKEN
         case .amountNotAllowedForTap:
             return CLEARENT_TAP_OVER_MAX_AMOUNT
         case .chipNotRecognized:
             return CLEARENT_CHIP_UNRECOGNIZED
         }
     }
     
-    static func info(for text: String) -> UserInfo? {
-        UserInfo.allCases.first { $0.description == text }
+    var description: String? {
+        if ClearentWrapper.shared.enableEnhancedMessaging && ClearentWrapper.shared.enhancedMessagesDict != nil {
+            guard let result = ClearentWrapper.shared.enhancedMessagesDict?[message] else { return nil }
+            return result == ClearentConstants.Messaging.suppress ? nil : result
+        }
+         return message
```

Comment:
If there is no value in dictionary for the current key, return `message`, not `nil`
```suggestion
        if ClearentWrapper.shared.enableEnhancedMessaging, let dict = ClearentWrapper.shared.enhancedMessagesDict, let result = dict[message] {
            return result == ClearentConstants.Messaging.suppress ? nil : result
        }
        return message
```